appinventor/components/src/com/google/appinventor/components/runtime/Pedometer.java

```diff
@@ -132,27 +137,6 @@ public void Reset() {
     startTime = System.currentTimeMillis();
   }
 
-  /**
```

Comment:
Any particular reason why these definitions were moved further down in the file? You could keep them here while replacing the body.

Reply:
While looking at `Pedometer.java` I noticed there was a section for deprecated properties and events. I thought I might as well at these new deprecated methods to this section. I have reversed the moving of the methods in the latest commit.